packages/clerk-js/src/core/clerk.ts

```diff
@@ -255,6 +256,25 @@ export default class Clerk implements ClerkInterface {
 
   public isReady = (): boolean => this.#isReady;
 
+  get allowedRedirectOrigins(): (string | RegExp)[] | undefined {
+    if (!this.#options.allowedRedirectOrigins) {
+      const origins = [];
+      if (inBrowser()) {
+        origins.push(window.location.origin);
+        origins.push(window.location.origin + '/*');
+      }
+
+      const frontendApi = this.frontendApi;
+
+      origins.push(`https://*.${getETLDPlusOneFromFrontendApi(frontendApi)}`);
+      origins.push(`https://*.${getETLDPlusOneFromFrontendApi(frontendApi)}/*`);
+
+      return origins;
+    }
+
+    return this.#options.allowedRedirectOrigins;
+  }
+
```

Comment:
This affects the public API of `window.Clerk`. Let's make it a util instead for now, as we haven't discussed making it public; we can always add it on `window.Clerk` later if required

Reply:
Done!